src/grisp_ir_drv.erl

```diff
@@ -0,0 +1,39 @@
+% @private
+-module(grisp_ir_drv).
```

Comment:
How are the messages sent? To the process calling `configure_ir/3`? Do we need an `grisp_ir` interface module for this functionality?

Reply:
The messages are sent to the caller, yes.

I thought about having a `grisp_ir` module and having a seperate process. But the purpose of all this is to get messages from the interrupt as soon as possible. So it must be the caller who is connected to the port. Also in case of any errors in the driver the caller might want to crash also.

Reply:
If the messages are sent to the port controlling process, yes. If they're sent to the caller of each command, you can still have a managing process running in the background. Some process has to own the port instance, and with this structure only one process can listen to interrupts in the whole system, right?

Reply:
> The Erlang process creating a port is said to be the port owner, or the connected process of the port. All communication to and from the port must go through the port owner. If the port owner terminates, so does the port (and the external program, if it is written correctly).
> -- <cite>http://erlang.org/doc/reference_manual/ports.html</cite>

In theory you could spawn multiple instances of the same driver. But I did not test at all whether this works.